webview-ui/src/i18n/locales/es/chat.json

```diff
@@ -205,6 +205,7 @@
 	"commandExecution": {
 		"running": "Corriendo",
 		"abort": "Abortar",
+		"running": "Ejecutando",
```

Comment:
It looks like it duplicated the "running" key here and maybe didn't update the runCommand.title?

Reply:
😓 , yup